packages/ember-htmlbars/lib/helper.js

```diff
@@ -0,0 +1,22 @@
+import Object from "ember-runtime/system/object";
+
+// Ember.Helper.extend({ compute(params, hash) {} });
+var Helper = Object.extend({
+  isHelper: true,
+  recompute() {
+    this._stream.notify();
+  }
+});
+
+// Ember.Helper.build(function(params, hash) {});
+Helper.reopenClass({
+  isHelperFactory: true,
+  build(helperFn) {
```

Comment:
Can you:

* rename this from `build` to `helper`
* move it out of `reopenClass` (because we don't extended class to have a `helper` on their factories)
* export a named `helper` function (for future ease in importing)